index.html

```diff
@@ -396,37 +498,55 @@ <h2>faqs</h2>
         </ul>
       </section>
     </main>
-    <footer>
-      <a href="#">BINABOX <img src="#" alt="Picture of logo" /></a>
-      <p>
+    <footer class="footer">
+      <a class="footer__logo logo logo--colorful" href="#">BINABOX <img src="#" alt="Picture of logo" /></a>
+      <p class="footer__desc">
         We are the best they to check rarity of <span> nft </span> collection
       </p>
-      <section>
-        <ul>
-          <li><a href="#">Subscribe</a></li>
-          <li><a href="#">Home</a></li>
-          <li><a href="#">Blog</a></li>
-          <li><a href="#">Catalog</a></li>
-          <li><a href="#">Contact us</a></li>
+      <section class="footer__details-wrapper">
+        <ul class="footer__list">
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Subscribe</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Home</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Blog</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Catalog</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Contact us</a></li>
         </ul>
-        <ul>
-          <li><a href="#">Company</a></li>
-          <li><a href="#">Terms Conditions</a></li>
-          <li><a href="#">Privacy Policy</a></li>
-          <li><a href="#">WhitePaper</a></li>
-          <li><a href="#">Help Center</a></li>
+        <ul class="footer__list">
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Company</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Terms Conditions</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Privacy Policy</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">WhitePaper</a></li>
+          <li class="footer__list-item">
+            <a class="footer__item-link" href="#">Help Center</a></li>
         </ul>
       </section>
-      <p>BINABOX 2022 - All right reserved</p>
-      <section>
-        <ul>
-          <li><a href="#">Twitter</a></li>
-          <li><a href="#">Facebook</a></li>
-          <li><a href="#">Telegram</a></li>
-          <li><a href="#">YouTube</a></li>
-          <li><a href="#">TikTok</a></li>
-          <li><a href="#">Reddit</a></li>
-        </ul>
+      <p class="footer__text">BINABOX 2022 - All right reserved</p>
+      <section class="footer__social-wrapper">
+        <nav class="footer__nav-social nav-social">
```

Comment:
I would suggest not using the nav tag here. Because the links don't lead to our site. About tag `nav` we can [read here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)